bluer/Cargo.toml

```diff
@@ -21,11 +22,12 @@ exclude = [
     "bluetooth-numbers-database/*.js",
     "bluetooth-numbers-database/*.json",
     "bluetooth-numbers-database/node_modules/*",
+    "meshd/*"
```

Comment:
Is meshd the example configuration for your examples? Then you should move it to the examples directory.